models/intermediate/int_iterable__list_user_unnest.sql

```diff
@@ -36,33 +73,44 @@ https://github.com/fivetran/dbt_iterable/issues/new/choose
         case when email_list_ids != '[]' then
         {% if target.type == 'snowflake' %}
         email_list_id.value
-        {% elif target.type == 'redshift' %}
-        json_extract_array_element_text(email_list_ids, cast(numbers.generated_number as {{ dbt.type_int() }}), true) 
-        {% else %} email_list_id
-        {% endif %}
-        else null end 
-        as 
-        email_list_id
+        {% else %} email_list_id {% endif %} else null end as email_list_id
 
     from user_history
 
-    cross join 
     {% if target.type == 'snowflake' %}
+    cross join 
         table(flatten(cast(email_list_ids as VARIANT))) as email_list_id 
```

Comment:
The original query here was always resulting in no results in the final model. I tinkered a bit and found that this did the trick! Let me know
```suggestion
    cross join 
        table(flatten(input => parse_json(email_list_ids))) as email_list_id
```